com.arviceblot.eso-addon-manager.json

```diff
@@ -0,0 +1,47 @@
+{
+    "app-id": "com.arviceblot.eso-addon-manager",
```

Comment:
`arviceblot.com` doesn't seem to indicate it's linked to this package. I get an error page.

Reply:
It is my personal domain, there's no real web content for this project outside the github repo. I have added a TXT entry `eso-addon-manager.arviceblot.com` for some proof.